wordpress.org/public_html/wp-content/plugins/official-wordpress-events/template-events.php

```diff
@@ -9,20 +9,26 @@
 		<ul class="ofe-event-list">
 			<?php foreach ( $day_events as $event ) : ?>
 				<li>
-					<?php if ( $event->location ) : ?>
-						<?php echo esc_html( 'online' === $event->location ? 'Online' : $event->location ); ?>
-						<span class="owe-separator"></span>
+					<?php // WordCamp.org doesn't collect start times, so only display those for meetups ?>
+						<?php if ( 'wordcamp' != $event->type ) : ?>
+							<?php echo date( 'g:i a', $event->start_timestamp ); ?>
```

Comment:
What do you think about testing for the presence of a valid timestamp, rather than the event type? We don't know yet if pilot events will provide a start time or not, so that'd be a bit more future proof.

Reply:
I'm curious as to why we didn't use `$event->timestamp` as the checking condition in the first place. Do you happen to know the history of that?

As I noticed that a WordCamp also has a time display, like in this example:
![image](https://github.com/WordPress/wordpress.org/assets/18050944/819f0717-36a2-4f2c-be63-c3bf66250dab)
But the time seems to be [incorrect](https://allevents.in/iloilo/wordcamp-iloilo-2023/200024454111504) in this case, which could possibly be one of the reasons why we initially didn't want to display the time for WordCamp?

There are also cases where the time display is [accurate](https://montclair.wordcamp.org/2023/schedule/), so it might just have been overlooked:
![image](https://github.com/WordPress/wordpress.org/assets/18050944/418c3a29-db27-43e2-993c-580f3655010e)

I think we should still be able to test for the presence of a valid timestamp and display the WordCamp time + other events time.

Reply:
Updated in d6e9deb40384c12f298243596fadaded2c4d2e56.